src/cli/utils/sharepoint.go

```diff
@@ -35,9 +40,22 @@ func ValidateSharePointRestoreFlags(backupID string, opts SharePointOpts) error
 		return errors.New("a backup ID is required")
 	}
 
-	// if _, ok := opts.Populated[FileCreatedAfterFN]; ok && !IsValidTimeFormat(opts.FileCreatedAfter) {
-	// 	return errors.New("invalid time format for created-after")
-	// }
+	if _, ok := opts.Populated[FileCreatedAfterFN]; ok && !IsValidTimeFormat(opts.FileCreatedAfter) {
+		fmt.Printf("What was I sent: %v\n", opts.FileCreatedAfter)
+		return errors.New("invalid time format for created-after")
```

Comment:
Better to reference the flag names here:
```suggestion
		return errors.New("invalid time format for " + utils.FileCreatedAfterFN)
```